scripts/cleanup-sam-resources.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# A script for cleaning up a user's Sam resources, such as those not cleaned
+# up after testing. This will attempt to delete all controlled (both shared and
+# private) resources owned by a user in Sam, as well as all their workspaces.
+#
+# This script will attempt to delete all of the resources and workspaces a user
+# has any access to, including ones the user does not have permission to delete
+# (such as those owned by other test users, where the specified user is only
+# a reader). This means some delete calls will fail with a 403 error and the
+# message "You may not perform any of [DELETE] on
+# {resource type}/{resource id}". This is expected and can be ignored.
+#
+# Input:
+#   access_token: An access token used for calls to Sam. This script will delete
```

Comment:
Where/how do I get this access token? It would be safer if we could supply the email, as that's unequivocally a test user.

Reply:
Either through service account impersonation (what we do for tests) or using login information from the test user tracker. It's probably possible to script the first one (assuming the user also renders configs separately) but I'm not sure how, and I think automating that would probably take more time than it saves.